Headers/DebugServer2/Target/ProcessBase.h

```diff
@@ -13,14 +13,14 @@
 
 #include "DebugServer2/Target/ProcessDecl.h"
 #include "DebugServer2/Target/ThreadBase.h"
+#include "DebugServer2/SoftwareBreakpointManager.h"
 
 #include <functional>
 #include <set>
 
 namespace ds2 {
 
-class BreakpointManager;
-class WatchpointManager;
+class HardwareBreakpointManager;
```

Comment:
You probably wanna remove this as well, and include the proper header after line 16 (`HardwareBreakpointManager.h`).